### ISSUE TITLE
InvalidArgumentException: Processors must be valid callables (callback or object with an __invoke method), BGalati\MonologSentryHandler\SentryHandler given

### ISSUE THREAD

**Issue by christheyounger** [B-Galati/monolog-sentry-handler#5]
HI there
I'm attempting to implement your solution as I share your feelings about the change to the implementation of the fingers crossed handler.
Here's what I've got:
```
$ravenHandler = new SentryHandler(new Hub($c->get('sentry')));
$logger->pushProcessor($ravenHandler);
```
Sentry is defined like this
```
return ClientBuilder::create(
                    $c->get('settings')['sentry'],
                )->getClient();
```
Still I'm getting this error
> InvalidArgumentException: Processors must be valid callables (callback or object with an __invoke method), BGalati\MonologSentryHandler\SentryHandler given

slim php 3.9
monolog v1.25.1

Are you able to shed any light on this?

https://github.com/brighte-capital/slim-core/pull/5

Thanks

**Comment by christheyounger** [comment]
I am a fool. I was using `pushProcessor` in place of `pushHandler`. Thank you sir.

**Comment by B-Galati** [comment]
Hello and thanks for this issue!

Beware of getsentry/sentry-php#878 (I will try to update the doc)
And also beware that the library might be broken for the new released version of the SDK: ^2.2 because of the hack needed for getsentry/sentry-php#811.

Cheers and let me know for anything!
